.eslintrc.json

```diff
@@ -2,7 +2,6 @@
   "extends": [
     "prettier"
   ],
-  "parser": "babel-eslint",
```

Comment:
This is a redundant requirement given the capabilities of eslint. It still exists in some of our other repos and should be removed if/when we find it. The accompanying babel-eslint devDependency package should also be removed from package.json.

Reply:
Will do, thanks!